src/url.js

```diff
@@ -36,6 +36,8 @@ const SERVING_TYPE_PREFIX = dict({
   'a': true,
   // Ad
   'ad': true,
+  // Actions viewer
+  'action': true,
```

Comment:
Why is this necessary? I don't see any usages of url.js functions in amp-viewer-assistance.js.

Reply:
You're right that we do not use `url.js` in `amp-viewer-assistance.js`. However 'action' was added to the  in order to whitelist 'action' as a valid prefix for [this assertion](https://github.com/ampproject/amphtml/blob/3c134ac06763ba485c8bf96c71799d96be563aca/src/url.js#L510).

Reply:
Can we extract this to a separate PR? I'm also curious what the specific codepath (which caller of `getSourceUrl()`) triggers that assertion for you.

Reply:
We can move this to a different PR, however it would be a breaking change. The codepath is whenever `getSourceUrl()` is called, most likely during [the open channel](https://github.com/ampproject/amphtml/blob/990fc5961efa6c5ed8affa7fd176ffa340277275/extensions/amp-viewer-integration/0.1/amp-viewer-integration.js#L170) call in `amp-viewer-integration.js`. It's necessary because the cdn url requests have an `action` prefix present.